src/Admin/AdminHelper.php

```diff
@@ -252,6 +230,33 @@ public function appendFormFieldElement(AdminInterface $admin, $subject, $element
             $newInstance = ObjectManipulator::addInstance($form->getData(), $associationAdmin->getNewInstance(), $fieldDescription);
 
             $associationAdmin->setSubject($newInstance);
+        } else {
+            // The `else` branch is executed when child form was not found (probably nested one).
+            // If `$childFormBuilder` was not found resulted in fatal error `getName()` method call on non object
+            // or if `$childFormBuilder` was found, but the admin object does not have a formFieldDescription with
+            // same name as `$childFormBuilder`.
+
+            $path = $this->getElementAccessPath($elementId, $subject);
+
+            $collection = $this->propertyAccessor->getValue($subject, $path);
+
+            if (!($collection instanceof Collection)) {
```

Comment:
IMO, we should avoid superfluous parentheses.

Reply:
Can we discuss this in dev-kit? 
So it's applied globally